.circleci/config.yml

```diff
@@ -0,0 +1,164 @@
+version: 2
+
+jobs:
+  checkout_code:
+    docker:
+      - image: jfullaondo/datadog-agent-builder-deb:0.0.2
```

Comment:
Do we have a plan for building and distributing the builder image? Shall we open-source the repo and build them in the docker hub for now?